### PR TITLE
Update user story template

### PR DESCRIPTION
**Pull request by axrolld**
The goal of this modification is the make US title shorter, transfering infos in the US description.

### CODE REVIEW

**Review by flomonster — DISMISSED** [review]

File: .github/ISSUE_TEMPLATE/user_story.md [hunk 1]
```diff
@@ -1,15 +1,27 @@
 ---
 name: User story
 about: Create a new user story for this project
-title: "[US] As [...], I'd like to [...], in order to [...]."
+title: "[US] [Name this US as it appears in the sprint goal]"
```
Comment (multun) [inline .github/ISSUE_TEMPLATE/user_story.md]: I don't think [US] is useful, there's already the actual kind:user-story tag, which enables filtering just by clicking on it

Comment (axrolld) [reply .github/ISSUE_TEMPLATE/user_story.md]: I know the tag allows us to filter the us among other issues, but I'd like to keep it to be able to identify them quickly, for example in the board, where tags are not visible.
Moreover [US] isn't really making the issue title much longer.
I don't care removing it but personnaly i find it quite useful.

Comment (axrolld) [reply .github/ISSUE_TEMPLATE/user_story.md:4]: For example in the board, where tags are not visible

Comment (flomonster) [reply .github/ISSUE_TEMPLATE/user_story.md]: I agree with the idea of removing `[US]` for consistency. We don't use a textual tag for other issues.

Comment (elzinko) [reply .github/ISSUE_TEMPLATE/user_story.md]: > I know the tag allows us to filter the us among other issues, but I'd like to keep it to be able to identify them quickly, for example in the board, where tags are not visible. Moreover [US] isn't really making the issue title much longer. I don't care removing it but personnaly i find it quite useful.

Managing simultaneously 2 ways of categorization is not a good practice. But I understand your point of view

Comment (axrolld) [reply .github/ISSUE_TEMPLATE/user_story.md]: Alright, i'll change it then

**Review by flomonster — APPROVED** [review]

**Pull request merged**